Client/mods/deathmatch/logic/CClientDFF.cpp

```diff
@@ -101,6 +101,21 @@ void CClientDFF::UnloadDFF()
 
 bool CClientDFF::ReplaceModel(unsigned short usModel, bool bAlphaTransparency)
 {
+    if (usModel >= CLOTHES_MODEL_ID_FIRST && usModel <= CLOTHES_MODEL_ID_LAST)
+    {
+        if (m_RawDataBuffer.empty() && m_bIsRawData)
+            return false;
+
+        if (m_RawDataBuffer.empty())
+        {
+            if (!FileLoad(std::nothrow, m_strDffFilename, m_RawDataBuffer))
+                return false;
+        }
+
+        g_pGame->GetRenderWare()->ClothesAddReplacement(m_RawDataBuffer.data(), usModel - CLOTHES_MODEL_ID_FIRST);
+        return true;
+    }
+
```

Comment:
Extract this method as `ReplaceClothes(uint32_t model)`
Call it in `DoReplaceModel` 
```c++
        if (CPlayerClothes::IsValidClothingModelId(usModel))
        {
            return ReplaceClothes(usModel);
        }
```